team-builder/src/components/TeamForm.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+
+const TeamForm = props => {
+  console.log("props", props);
+  const [team, setTeam] = useState({ name: "", email: "", role: "" });
+
+  const handleChanges = e => {
+    // console.log(team);
```

Comment:
Please remove any zombie code before committing.

Zombie code = commented out code.

Comments are used to explain things to other developers and you if you come back to this project, not to preserve code. You should be committing changes frequently enough that you do not lose code that you delete when it is not needed.